tests/snapshots/repo__repo.snap

```diff
@@ -7,8 +7,8 @@ expression: info
   "gitVersion": "git version",
   "project": {
     "repo_name": "repo",
-    "number_of_branches": 0,
-    "number_of_tags": 1
+    "number_of_branches": "0",
+    "number_of_tags": "1"
```

Comment:
I guess a change in JSON types makes this a breaking change :thinking:

Reply:
Thinking about this some more, maybe the format should only affect how the stats are *printed*, but keep the struct's types the same, so data outputs don't change their type.